envi/tests/test_arch_arm.py

```diff
@@ -1870,14 +2125,51 @@ def validateEmulation(self, emu, op, setters, tests, tidx=0):
 
         return success
 
+import re
```

Comment:
no bitrot here, `cvtImmToHex()` is used in the unittests.

i'm ok with moving some of this to another location.  let's do it in it's own PR, and i'd like to agree upon a location to move them to before submitting that PR.  let's talk in DM.  hopefully this won't hold up this PR, since these test generators have been here since 2017